test/test_fx.py

```diff
@@ -2634,6 +2634,7 @@ def test_get_torch_func_signature_exhaustive(self, device, dtype, op):
                            'einsum',
                            'expand',
                            'expand_as',
+                           'fill_',
```

Comment:
Just curious, @kshitij12345 - what is this test for? (to figure out if `zero_` needs to be added as well).

Reply:
I am not sure what the test does, but if it fails when you run it then you must add the operator to the skip list. 
Note:
This test doesn't work with lambda so if you use, lambda to wrap your operator then this will fail.